src/sentry/integrations/github/webhook.py

```diff
@@ -173,19 +175,6 @@ def _handle_delete(self, event: Mapping[str, Any], integration: Integration) ->
         ).update(status=ObjectStatus.DISABLED)
 
 
-class InstallationRepositoryEventWebhook(Webhook):
```

Comment:
This webhook was declared here and on the Github Enterprise integration and not implemented for about five years. If we revamp the integration and add new webhooks to consume, we can address that then.